### PR TITLE
Change default webUI port back to 5001

### PR DESCRIPTION
**Pull request by justinclift**
## What type of PR is this? 

- [x] Bug Fix

## Description

This PR changes the default (tcp) port for the web user interface back to port 5001.

The recent change to port 5000 (to match an old default) turned out to be more painful than it's worth.

So, lets keep using port 5001 after all. :smile:

## How is this tested?

- [x] Unit tests (pytest, jest)
- [x] E2E Tests (Cypress)

### CODE REVIEW

**Comment by justinclift** [comment]
@eradman @AndrewChubatiuk FYI.  Am running the tests manually on my local PC, but figured we might as well run them on our CI at the same time. :smile:

**Comment by justinclift** [comment]
Hmmm, this is badly failing on the Cypress tests on my local system.  Looks like I missed something.

@AndrewChubatiuk Any ideas?

**Comment by justinclift** [comment]
Looks like my local Cypress testing isn't starting all of the desired workers containers.

File: compose.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ services:
       - worker
       - scheduler
     environment:
-      CYPRESS_baseUrl: http://server:5000
+      CYPRESS_baseUrl: http://server:5001
```
Comment (AndrewChubatiuk) [inline compose.yaml:69]: no need to make this change. cypress will be using docker network and in docker network redash server is on port 5000

Comment (justinclift) [reply compose.yaml:69]: Thanks.  I'll push a follow up commit in a few minutes.  Just want to see if the Cypress tests get further with the current commit first.  In my local testing they're now passing. :smile:

Comment (justinclift) [reply compose.yaml]: Pushed a commit to change that back to 5000.

**Comment by justinclift** [comment]
It's passing the CI tests now too, so I'm just going to merge it.

Thanks for the assist with this too @AndrewChubatiuk. :smile:

**Pull request merged**